src/main/java/javax/measure/UnitConverter.java

```diff
@@ -53,35 +54,50 @@
 
     /**
      * Indicates if this converter is an identity converter. The identity converter returns its input argument ({@code convert(x) == x}).
+     * <p>
+     * Note: Identity converters are also always 'linear'. {@link UnitConverter#isLinear()}
+     * </p>
      *
      * @return {@code true} if this converter is an identity converter.
      */
     boolean isIdentity();
 
     /**
-     * A linear transformation between two vector spaces V and W is a map T : V -> W such that the following hold:
+     * Indicates whether this converter represents a (one-dimensional) linear transformation, that is
+     * a <a href="https://en.wikipedia.org/wiki/Linear_map">linear map (wikipedia)</a> from a one-dimensional 
+     * vector space (a scalar) to a one-dimensional vector space. Typically from 'R' to 'R', with 'R' the 
+     * real numbers.  
+     * 
+     * <p>
+     * Given such a 'linear' converter 'A', let 'u', 'v' and 'r' be arbitrary numbers, then the following 
+     * must hold by definition: 
      *
      * <ul>
-     * <li>{@code convert(u + v) == convert(u) + convert(v)}</li>
-     * <li>{@code convert(r * u) == r * convert(u)}</li>
+     * <li>{@code A(u + v) == A(u) + A(v)}</li>
+     * <li>{@code A(r * u) == r * A(u)}</li>
      * </ul>
+     * </p>
      *
-     * A linear transformation may or may not be injective or surjective.
-     * When V and W have the same dimension, it is possible for T to be invertible, meaning there exists a T⁻¹ such that
-     * TT⁻¹ = I. It is always the case that  T(0)=0.
-     * Also, a linear transformation always maps lines to lines (or to zero).
+     * <p>
+     * Given a second 'linear' converter 'B', commutativity of composition follows by above definition:
      *
+     * <ul>
+     * <li>{@code (A o B) (u) == (B o A) (u)}</li>
+     * </ul>
+     * 
+     * In other words, two 'linear' converters do have the property that {@code A(B(u)) == B(A(u))}, meaning 
+     * for 'A' and 'B' the order of their composition does not matter. Expressed as Java code:
+     * </p>
      * <p>
-     * For linear converters the following property always hold:
+     *{@code A.concatenate(B).convert(u) == B.concatenate(A).convert(u)}
+     * </p>
+     * <p>
+     * Note: For composing UnitConverters see also {@link UnitConverter#concatenate(UnitConverter)}.
      * </p>
      *
```

Comment:
Should we add "Note that <cite>linear transformation</cite> are not necessarily the same as <cite>linear function</cite>."? The fact that I interpreted this method as "is linear function", that is as an equation of the form "y = a*x + b" was the reason for my initial objection against the method name (the interpretation as "linear map" instead resolve that objection).

Reply:
Do you feel, that the proposed javadoc does still leave any room for misinterpretation?

Reply:
I think the proposed javadoc is clear, To be frank, I was not aware that "linear map" and "linear function" could differ before this link to wikipedia, and my lack of knowledge about this difference caused me to react against the `isLinear` method name. I though that some other peoples may fall in the same trap, but I have no strong opinion - it is fine for me to leave the javadoc as is.

Reply:
Ok, thx. No worries, I was confused as well. But I think our definition is bulletproof, that's what matters here.